src/client/datascience/notebook/types.ts

```diff
@@ -12,7 +12,9 @@ import {
 export const INotebookContentProvider = Symbol('INotebookContentProvider');
 
 export const INotebookKernelProvider = Symbol('INotebookKernelProvider');
-export interface INotebookKernelProvider extends NotebookKernelProvider {}
+export interface INotebookKernelProvider extends NotebookKernelProvider {
+    onDidGetFontAwesomeMessage: vscode.Event<{ webview: NotebookCommunication; message: any }>;
```

Comment:
The name here seems a bit odd. Maybe 'onDidGetMessage'? The notebook kernel provider doesn't need to know which message, right?